applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_damped_U_Pw_scheme.hpp

```diff
@@ -118,7 +117,7 @@ class NewmarkQuasistaticDampedUPwScheme : public NewmarkQuasistaticUPwScheme<TSp
                          const ProcessInfo& CurrentProcessInfo)
     {
         // adding damping contribution
-        if (C.size1() != 0) noalias(LHS_Contribution) += (mGamma/(mBeta*mDeltaTime))*C;
+        if (C.size1() != 0) noalias(LHS_Contribution) += (mGamma/(mBeta*this->GetDeltaTime()))*C;
```

Comment:
This looks as VELOCITY_COEFFICIENT defined in newmark_quasistatic_U_Ps_scheme.hpp. Can such a coefficient be created and used here?